plugins/mysql/mysql_audit

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
+# -*- python -*-
+
+"""
+=head1 INTRODUCTION
+
+Plugin to monitor the MySQL audit log connection count
+
+=head1 INSTALLATION
+
+Place in /etc/munin/plugins/ (or link it there using ln -s)
+
+=head1 CONFIGURATION
+
+Add this to your /etc/munin/plugin-conf.d/munin-node:
+
+=over 2
+
+      [mysql_audit]
+      user mysql
+      period 3
+      logfile /var/log/mysql/audit.log
+      sorted 1 # sort output (if not: unsorted)
+      toplist 10 # only the top 10 (if unset: all of them). implies sorted
+
+=back
+
+NOTE: MySQL needs to be configured manually in order to enable the audit log!
+This can be done as follows:
+
+=over 2
+
+      plugin-load = server_audit=server_audit.so
+      server_audit_events = connect
+      server_audit_output_type=file
+      server_audit_file_path = /var/log/mysql/audit.log
+      server_audit_file_rotate_size = 512M
+      server_audit_logging = ON
+      server_audit_file_rotations = 5
+
+=back
+
+=head1 HISTORY
+
+2017-11-03: v 1.0 Bert Van de Poel <bert@bhack.net>: created
+2020-07-19: v 1.1 pcy <pcy@ulyssis.org>: added config options
+
+=head1 USAGE
+
+Parameters understood:
+
+  config   (required)
+  autoconf (optional - used by munin-config)
+
+=head1 MAGIC MARKERS
+
+#%# family=auto
+#%# capabilities=autoconf
+"""
+
+
+import sys
+import os
+from datetime import datetime, timedelta
+import operator
+
+
+def weakbool(x):
+    return x.lower().strip() in {'true','yes','y','1'}
```

Comment:
PEP8 welcomes spaces between operators (`,`). `flake8` is your friend :)

Reply:
Yes. The CI pipeline actually runs `flake8`, which is how I fixed a few of these issues before, but it always choked on the magic markers (cf. [E265](https://www.flake8rules.com/rules/E265.html)), which is why I added [this directive](https://github.com/munin-monitoring/contrib/blob/9ad2de67c790d01d8c872ae70110ee89be979789/plugins/mysql/mysql_audit#L160) at the very end of the file. But it looks like `flake8` misparses that and starts ignoring *everything*...

Reply:
Good catch! :)

Indeed the flake8 documentation says, that files containing the line `# flake8: noqa` are skipped. It seems like a partial match (not the full line) is sufficient for this.

Probably `E265` is not relevant anymore, since you switched to the perldoc header?

I just removed the line from `mysql_disk_by_prefix`. Please do the same for your open pull requests, if applicable.